projects/admin/src/app/record/detail-view/document-detail-view/document-detail-view.component.ts

```diff
@@ -202,4 +204,55 @@ export class DocumentDetailViewComponent implements DetailRecord, OnInit, OnDest
     }
     return 'standard';
   }
+
+  /**
+   * Get "part of" information
+   *
+   * @return; part of to display
+   */
+  get partOf(): any {
+    const data = this.record.metadata.partOf;
+    return data;
+  }
+
+  /**
+   * Format "part of" numbering for display
+   *
+   * @param num: numbering to format
+   * @return formatted numbering (example: 2020, vol. 2, nr. 3, p. 302)
+   */
+  formatNumbering(num: any) {
+    const numbering = [];
+    if (num.year) {
+      numbering.push(num.year);
+    }
+    if (num.volume) {
+      const volume = [this._translateService.instant('vol'), num.volume];
+      numbering.push(volume.join('. '));
+    }
+    if (num.issue) {
+      const issue = [this._translateService.instant('nr'), num.issue];
+      numbering.push(issue.join('. '));
+    }
+    if (num.pages) {
+      const pages = [this._translateService.instant('p'), num.pages];
+      numbering.push(pages.join('. '));
+    }
+    return numbering.join(', ');
+  }
+
+  /**
+   * Get "part of" label from host document type
+   * @param hostDocument - host document
+   * @return corresponding translated label
+   */
+  getPartOfLabel(hostDocument: any) {
+    if (hostDocument.metadata.issuance.subtype === 'periodical') {
+      return this._translateService.instant('Journal');
+    }
+    if (hostDocument.metadata.issuance.subtype === 'monographicSeries') {
+      return this._translateService.instant('Series');
+    }
+    return this._translateService.instant('Published in');
```

Comment:
Did you forget to extract strings ? I don't find the string 'Published in' nor on UI, nor on Backend

Reply:
Good you reported that. I was waiting to be back on mac to do that.

Reply:
Why you don't use switch in this part ?